charts/spire/charts/spire-server/values.yaml

```diff
@@ -909,6 +909,10 @@ experimental:
   enabled: false
   ## @param experimental.cacheReloadInterval The amount of time between two reloads of the in-memory entry cache.
   cacheReloadInterval: 5s
+  ## @param experimental.eventsBasedCache Use events to update the cache with what's changed since the last update.
+  eventsBasedCache: false
+  ## @param experimental.pruneEventsOlderThan How old an event can be before being deleted. Used with events based cache.
+  pruneEventsOlderThan: 12h
```

Comment:
https://github.com/spiffe/spire/blob/main/doc/spire_server.md lists the default as 5s.

Reply:
the cache reload default is 5s. Pruning events is 12 hours.

Reply:
Doh. Your right. Sorry about that.